addons/l10n_my_edi_extended/models/account_move.py

```diff
@@ -0,0 +1,188 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import time
+from collections import defaultdict
+
+import werkzeug
+
+from odoo import fields, models, api, _, SUPERUSER_ID
+from odoo.exceptions import UserError
+from odoo.tools import image_data_uri
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    l10n_my_edi_invoice_long_id = fields.Char(
+        string="MyInvois Long ID",
+        copy=False,
+        readonly=True,
+        export_string_translation=False,
```

Comment:
Why not?

I'd understand for the field below, but this one could be exported etc, no?